lib/threads/catalog.py

```diff
@@ -15,15 +16,19 @@ def __init__(self):
         self.load_palettes(self.get_palettes_path())
 
     def get_palettes_path(self):
-        if getattr(sys, 'frozen', None) is not None:
-            path = os.path.join(sys._MEIPASS, "..")
-        else:
-            path = dirname(dirname(dirname(realpath(__file__))))
+        path = guess_inkscape_config_path()
+
+        # Fallback on inkstitch palettes folder
+        if not os.path.exists(path):
```

Comment:
The inkscape palettes directory might exist even if the user didn't install Ink/Stitch palettes.  That would make Ink/Stitch unable to recognize thread color names.  Maybe we can use palette files from both places?

Reply:
Hm, what would you think, if we checked if there is any InkStitch file in the folder and go on with either the one path or the other?

Reply:
I'm still not sure.  What if our French friend has their one palette that they made, but then they also want to open an SVG created by someone else?

Reply:
Ok, convinced ;) Updated the code to loop through both.

Reply:
Looks great!

Reply:
Let's not have every palette being doubled in the list if the add-ons are installed. It would crowd up the list in the print settings, when you want to set it manually.

Reply:
Good point!